lib/dune

```diff
@@ -1,4 +1,4 @@
 (library
   (name ocaml_ci)
   (preprocess (pps ppx_deriving.std ppx_deriving_yojson))
-  (libraries logs current current_github current.cache ppx_deriving_yojson.runtime))
+  (libraries logs current current_github current.cache dockerfile ppx_deriving_yojson.runtime))
```

Comment:
`service/dune` now no longer depends on `dockerfile`; you should remove it there,

Reply:
Done